### PR TITLE
Add GitHub Action workflow (x64) and Travis (arm64) build 

### PR DESCRIPTION
**Pull request by xerial**
- Use GitHub Actions for regular CIs
- Use Travis CI for testing on Arm64 with openjdk11 https://docs.travis-ci.com/user/multi-cpu-architectures/ It's an emulated environment running on x64 machine.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: jcheckstyle
+        run: ./sbt jcheckStyle
+  test_jdk11:
+    name: Test JDK11
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
```
Comment (yuokada) [inline .github/workflows/CI.yml:32]: If the AdoptJDK installation is supported in setup-java action, it's better to replace with it.
https://github.com/actions/setup-java

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        with:
+          java-version: adopt@1.11
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache
+          key: ${{ runner.os }}-jdk11-${{ hashFiles('**/*.sbt') }}
+          restore-keys: ${{ runner.os }}-jdk11-
```
Comment (yuokada) [inline .github/workflows/CI.yml:39]: Is this key correct?

File: .github/workflows/CI.yml (hunk 1 above)
Comment (xerial) [reply .github/workflows/CI.yml:32]: msgpack-java's test is written in Scala, and also setup-scala can install natively-built sbt (csbt) using GraalVM. So let me keep using setup-scala.

File: .github/workflows/CI.yml (hunk 2 above)
Comment (xerial) [reply .github/workflows/CI.yml:39]: Yes. It has fallback mechanism https://circleci.com/docs/2.0/caching/#example-caching-configuration

Comment (yuokada) [reply .github/workflows/CI.yml:39]: I read below document after I wrote above comment. And though I understood the specification of `restore-keys`, I forgot removing it.
https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

**Pull request merged**